### DISCUSSION TITLE
Welcome to Chromecast-Kiosk Discussions!

### DISCUSSION THREAD

**Discussion by mrothenbuecher** [mrothenbuecher/Chromecast-Kiosk#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MTechH7** [comment]
Hi 

Hope you are well i am interested in this project 
However i do not have much knowledge when it comes it setting them up 
Could you assist in possibly doing a quick video tutorial ?

Regards
Mo

**Comment by whill121980** [comment]
I was able to get this set up and working properly with Google Slides. Thank you for the quality product! Any advice on getting this working with PowerPoint presentations using an embed link for the PowerPoint 365 web app? Thanks!

**Reply by MTechH7** [reply]
Hi William would you be able to assist in creating a video as to how you set this up as I am abit new to this but struggling

**Reply by whill121980** [reply]
> I was able to get this set up and working properly with Google Slides. Thank you for the quality product! Any advice on getting this working with PowerPoint presentations using an embed link for the PowerPoint 365 web app? Thanks!

To update here, I have successfully gotten the PowerPoint presentation to cast. However, auto-advance isn't working and there is a navbar on the bottom of the screen.

**Reply by whill121980** [reply]
> Hi William would you be able to assist in creating a video as to how you set this up as I am abit new to this but struggling

Sorry, I don't have time to make a video about it at this time.
